### PR TITLE
WCS issues fixed

### PR DESCRIPTION
**Pull request by laldoroty**
@wmwv requesting review. I want to merge this and delete the branches so I can clean up the mess I made with the WCS issue and all the wrong branches and commits and such. But this checkpoint runs. 

note wrong branch title, this should have been deleted

### CODE REVIEW

File: sfft_and_animate.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def sfft_and_animate(oid,band):
     # First, need all the images that don't contain the SN. 
     out_tab = get_mjd_info(start,end,return_inverse=True)
     out_rows = np.where(np.isin(tab['pointing'],out_tab['pointing']))[0]
-    out_tab = tab[out_rows][:10] # Only use the first 5 images 
+    out_tab = tab[out_rows][:5] # Only use the first 5 images
```
Comment (wmwv) [inline sfft_and_animate.py:83]: Why 5, or 10 images?

File: sfft_and_animate.py [hunk 2]
```diff
@@ -91,15 +91,21 @@ def sfft_and_animate(oid,band):
...
         skysub = sky_subtract(path=path)
         skysub_ref_init.append(skysub)
-        align = imalign(template_path=skysub_ref_init[0],sci_path=skysub)
+        align = imalign(template_path=skysub_ref_init[0],sci_path=skysub,force=True)
         imalign_ref_init.append(align)
 
         # Check that each reference tile contains an overlap_size cutout over the SN location.
         # This prevents PSF discontinuities too close to the SN. 
         with fits.open(align) as hdu:
+            print('ALIGN_PATH')
```
Comment (wmwv) [inline sfft_and_animate.py:100]: Someday wrap these in a `debug` or `verbose` for the logging.

File: sfft_and_animate.py [hunk 3]
```diff
@@ -182,28 +197,34 @@ def sfft_and_animate(oid,band):
...
-        diff, soln = difference(scipath, refpath, sci_psf_path, coadd_psf_path, backend='Numpy')
-        dcker_path = decorr_kernel(scipath, refpath, sci_psf_path, coadd_psf_path, diff, soln)
+        diff, soln = difference(scipath, refpath, sci_psf_path, coadd_psf_path, backend='Numpy', force=True)
+        print('DIFFERENCE IMAGE PATH')
+        print(diff)        
+        # NOTE: Decorrelation kernel only uses the science and reference images to calculate
+        # sky background, so don't use the cross-convolved images. Use the sky-subtracted ones.
+        # (And in the case of the science image, the aligned and sky-subtracted one.)
+        dcker_path = decorr_kernel(sci_imalign_path, ref_4k, sci_psf_path, coadd_psf_path, diff, soln)
+        print('DECORR KERNEL PATH')
```
Comment (wmwv) [inline sfft_and_animate.py:223]: Wrap these in some sort of `debug`, `verbose`, or loggin

**Review by wmwv — APPROVED** [review]
Can you think of a test that quickly demonstrates the fix and will prevent it from re-occuring?

File: sfft_and_animate.py (hunk 1 above)
Comment (laldoroty) [reply sfft_and_animate.py:83]: Ah, I took it down to 5 for speed while testing. No particular logic.

File: sfft_and_animate.py (hunk 2 above)
Comment (laldoroty) [reply sfft_and_animate.py:100]: Yes, definitely, for now they are intended as debugging statements but wrapping them into verbose is a good idea.

**Pull request merged**